src/components/fields/checkbox-group/checkbox-group.tsx

```diff
@@ -87,6 +87,7 @@ export const CheckboxGroup = (props: IGenericFieldProps<ICheckboxGroupSchema>) =
 						<StyledCheckbox
 							{...otherSchema}
 							data-testid={TestHelper.generateId(id, "checkbox")}
+							aria-label={option.label}
```

Comment:
generate a unique id for each radio button and update <Label htmlFor={id} />

this should provide the associated label without specifying aria-label